docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst

```diff
@@ -0,0 +1,83 @@
+or a deficiency of hemoglobin in the blood. Anemia is typically classified by
+hemoglobin concentrations below a defined threshold that varies by age and sex.
+Severity of anemia is similarly classified according to ranges of hemoglobin
+concentrations. Anemia is associated with increased morbidity and mortality and
+symptoms of anemia often include weakness, fatigue, and difficulty
+concentrating [Kassebaum-et-al-2016]_.
+
+ Notably, anemia may be caused by many diverse factors. Examples of factors
+ that may cause anemia include genetic mutations in hemoglobin genes, acute or
+ chronic blood loss, altered red blood cell morphology, inadequate nutritional
+ intake, and others [Kassebaum-et-al-2016]_.
+
+**Iron deficiency anemia** is a type of anemia that is due to insufficient
+iron levels, which lead to a deficiency of hemoglobin in the blood. Notably,
+iron deficiency anemia can occur when dietary intake of iron is insufficient or when there is excess loss of iron from the body (ex: menstrual disorders, hookworm disease, etc.). Iron deficiency anemia is the most common cause of anemia globally in most populations and will respond to iron supplementation.
+
+**Dietary iron deficiency anemia** is a specific type of iron deficiency anemia
+that is due to inadequate dietary intake of iron, leading to inadequate iron
+levels in the body and a subsequent deficiency of hemoglobin in the blood.
+
+GBD 2019 Hemoglobin, Anemia, and Iron Deficiency Overview
+---------------------------------------------------------
+
+The hemoglobin, anemia, and iron deficiency estimates in GBD 2019 are complex and related. This section will provide a brief overview of the different components relating to these topics in GBD 2019 for context.
+
+Hemoglobin Model
+^^^^^^^^^^^^^^^^
+
+The hemoglobin model in GBD 2019 is the starting point for both the anemia impairment and iron deficiency risk factor models in GBD 2019. The hemoglobin model estimates the *distribution* of population hemoglobin concentration for each location-, age-, sex-, and year-specific demographic group. This is done by estimation of mean and standard deviation hemoglobin values and a corresponding ensemble distribution fit. 
+
+This model will allow the assignment of specific hemoglobin concentration values to indiidual simulants and will likely need to be used in order to model either the anemia impairment or iron deficiency risk factor.
+
+:ref:`The 2019 GBD Hemoglobin Model Documentation Page can be found here <2019_hemoglobin_model>`.
+
+Anemia Impaiment
+^^^^^^^^^^^^^^^^
+
+The anemia impairment model in GBD 2019 uses the hemoglobin distribution model to evaluate the total prevalence of anemia ("anemia envelope") based on age- and sex-specific thresholds. The anemia impairment model also performs *causal attribution* in which specific causes of anemia (for example, dietary iron deficiency or maternal hemorrhage) are assigned to all of the cases in the anemia envelope. 
```

Comment:
"are assigned to all of the cases in the anemia envelope."

Is this supposed to say cases or causes?

Reply:
Cases. As in a cause is assigned to each "case" of anemia, where case is individuals with anemia. I can add some clarity there.